person.rb

```diff
@@ -1,4 +1,6 @@
-class Person
+require_relative './nameable'
+
+class Person < nameable
```

Comment:
- [ ] I guess you made a typo here because you are inheriting from a class that doesn't exist. `Nameble` with a capital first letter should be the name of the class.

Error Screenshot |
:--------------------:|
<img width="1097" alt="Screen Shot 2022-11-11 at 8 55 39 PM" src="https://user-images.githubusercontent.com/39523312/201429976-3efebc2b-b22f-4a2c-b6cd-bf9b28dddd3d.png"> |


Because of this error, I was unable to test the other functionalities of your project. I would suggest you take a look at it.